content/docs/learning-guide/tutorials/assets/physx-colliders.md

```diff
@@ -0,0 +1,162 @@
+
+{{< note >}}
+If you intend for the entity to be static, you should also activate the **Static** property in the **Transform** component of the entity. The **Static** property optimizes the entity, ensuring the best runtime performance for static entities.
+{{< /note >}}
+
+10. If you intend to create a kinematic or dynamic entity, you must add a **PhysX Rigid Body** component as well. With the entity selected, in Entity Inspector, choose **Add Component** and select PhysX Rigid Body from the component list. Ensure the **Static** property in the entity's Transform component is not active.
+
+{{< note >}}
+Adding the PhysX Rigid Body component makes the entity dynamic. If you choose the {{< icon simulate-physics.svg >}} simulation button, the entity will drop with gravity.
+
+If you want to create a kinematic entity that is animated through script, activate the **Kinematic** property in the PhysX Rigid Body component.
+{{< /note >}}
+
+## Decompose input meshes
+
+Assets that are composed of multiple meshes, or assets that have complex meshes, might require similarly complex PhysX collider assets. This is particularly true for kinematic and dynamic entities where triangle mesh colliders can't be used. In these scenarios, the input mesh can be decomposed into convex parts. Primitive colliders or convex colliders can be automatically generated and fitted to each part and processed as collider `.pxmesh` product assets.
+
+{{< note >}}
+Mesh decomposition doesn't alter the input mesh. Mesh decomposition is part of the process of generating and fitting collider assets. Mesh decomposition can only be used with primitive or convex colliders.
+
+Mesh decomposition in O3DE uses the [V-HACD Library](https://github.com/kmammou/v-hacd). Refer to the preceding link for more information and example images demonstrating decomposition of complex meshes.
+{{< /note >}}
+
+To use mesh decomposition, do the following:
+
+1. In Scene Settings, in the PhysX mesh group, choose either `Primitive` or `Convex` as the **Export As** property. 
+1. Activate the **Decompose Meshes** option.
+
+Activating Decompose Meshes reveals many options you can use to fine tune mesh decomposition. For guidance on mesh decomposition options, refer to the [Decompose Meshes](/docs/user-guide/assets/scene-settings/physx-tab/#decompose-meshes) topic in the User Guide.
+
+### Primitive decomposition
+
+When Decompose meshes is activated for primitive colliders, the input mesh is decomposed into convex parts, the best fitting primitive shapes are automatically selected and transformed to encompass each part, and the primitives are processed as a collider `.pxmesh` product asset. In the example below, the input mesh (left) is decomposed in to 6 parts and a primitive collider is automatically fitted to each part (right). Most parts are fitted with box primitive colliders, but one part, near the bottom of the asset, has a capsule primitive collider.

```

Comment:
```suggestion
When you enable Decompose Meshes for primitive colliders, the input mesh decomposes into convex parts, O3DE automatically selects the best-fitting primitive shapes and transforms them to encompass each part, and O3DE processes primitives as a collider `.pxmesh` product asset. In the following image, the input mesh (left) decomposes into six parts, and O3DE automatically fits a primitive collider to each part (right). Most parts are fitted with box primitive colliders. However, one part near the bottom of the asset has a capsule primitive collider.
```
Misc. edits similar to previous edits in this topic. Be sure to confirm that O3DE is the correct subject for these attempts to reword passive voice construction.